messages/protos/messages.proto

```diff
@@ -755,6 +801,11 @@ message PusherToBackMessage {
     AskPositionMessage askPositionMessage = 22;
     EditMapCommandMessage editMapCommandMessage = 23;
     PingMessage pingMessage = 24;
+    KickOffUserMessage kickOffUserMessage = 25;
+    MuteParticipantIdSpaceMessage muteParticipantIdSpaceMessage = 26;
+    MuteEveryBodySpaceMessage muteEveryBodySpaceMessage = 27;
+    MuteVideoParticipantIdSpaceMessage muteVideoParticipantIdSpaceMessage = 28;
+    MuteVideoEveryBodySpaceMessage muteVideoEveryBodySpaceMessage = 29;
```

Comment:
All those messages are called "SpaceMessage" but they do not apply to spaces, right?
You should rename those (remove the "space" word everywhere when it is a direct message that applies to groups)